reqhandler/request_handler.go

```diff
@@ -21,3 +22,34 @@ func readRequest(udpConn net.PacketConn) ([]byte, error) {
 	_, _, err := udpConn.ReadFrom(b)
 	return b, err
 }
+
+func extractID(d []byte) uint16 {
+	return (uint16(d[0]) << 8) | uint16(d[1])
```

Comment:
Hmm. I think there is a miscommunication. I am not asking to make the check at this level. I don't yet see check for the length anywhere. That is what I am addressing.